paddle/cinn/ir/ir.cc

```diff
@@ -268,6 +268,27 @@ Expr For::Make(Var loop_var,
 
   return Expr(node);
 }
+Expr For::Make(Var loop_var, Expr min, Expr extent, Expr body) {
+  auto node = make_shared<For>();
+  CHECK(loop_var.defined());
```

Comment:
Please add message for CHECK failure. Now we would like every error message can be user-friendly.

Reference: https://github.com/PaddlePaddle/Paddle/wiki/Paddle-Error-Message-Writing-Specification

Reply:
Done